pkgs/development/libraries/qt-6/qtModule.nix

```
@@ -32,9 +38,48 @@ stdenv.mkDerivation (args // {
         moveToOutput "$dir" "$dev"
       done
     fi
+    fixQtBuiltinPaths $out/lib "*.pr?"
     ${args.postInstall or ""}
   '';
 
+  preConfigure = ''
+    ${args.preConfigure or ""}
+
+    fixQtBuiltinPaths . '*.pr?'
+  '' + lib.optionalString (builtins.compareVersions "5.15.0" version <= 0)
+  # Note: We use ${version%%-*} to remove any tag from the end of the version
+  # string. Version tags are added by Nixpkgs maintainers and not reflected in
+  # the source version.
+  ''
+    if [[ -z "$dontCheckQtModuleVersion" ]] \
+        && grep -q '^MODULE_VERSION' .qmake.conf 2>/dev/null \
+        && ! grep -q -F "''${version%%-*}" .qmake.conf 2>/dev/null
+    then
+      echo >&2 "error: could not find version ''${version%%-*} in .qmake.conf"
+      echo >&2 "hint: check .qmake.conf and update the package version in Nixpkgs"
+      exit 1
+    fi
+
+    if [[ -z "$dontSyncQt" && -f sync.profile ]]; then
+      ${self.qtbase.dev}/libexec/syncqt.pl -version "''${version%%-*}"
+    fi
+  '';
+
+  postFixup = ''
+    if [ -d "''${!outputDev}/lib/pkgconfig" ]; then
+        find "''${!outputDev}/lib/pkgconfig" -name '*.pc' | while read pc; do
+            sed -i "$pc" \
+                -e "/^prefix=/ c prefix=''${!outputLib}" \
+                -e "/^exec_prefix=/ c exec_prefix=''${!outputBin}" \
+                -e "/^includedir=/ c includedir=''${!outputDev}/include"
+        done
+    fi
```

```suggestion
    if [ -d "''${!outputDev}/lib/pkgconfig" ]; then
      find "''${!outputDev}/lib/pkgconfig" -name '*.pc' | while read pc; do
        sed -i "$pc" \
          -e "/^prefix=/ c prefix=''${!outputLib}" \
          -e "/^exec_prefix=/ c exec_prefix=''${!outputBin}" \
          -e "/^includedir=/ c includedir=''${!outputDev}/include"
      done
    fi
```
we use 2 spaces in nix files and only 4 spaces in shell only files.

---

fixed here, haven't changed in the qt5 version of the same file

---

Thats fine.